src/esp/geo/CoordinateFrame.h

```diff
@@ -11,50 +11,49 @@
 namespace esp {
 namespace geo {
 
+namespace Mn = Magnum;
+
 //! Represents a frame of reference defined by an origin and
 //! two semantically meaningful directions: "up" and "front", or
 //! equivalently "gravity" and "back"
 class CoordinateFrame {
  public:
-  explicit CoordinateFrame(const vec3f& up = ESP_UP,
-                           const vec3f& front = ESP_FRONT,
-                           const vec3f& origin = vec3f::Zero());
-  explicit CoordinateFrame(const quatf& rotation,
-                           const vec3f& origin = vec3f::Zero());
+  explicit CoordinateFrame(const Mn::Vector3& up = ESP_UP,
+                           const Mn::Vector3& front = ESP_FRONT,
+                           const Mn::Vector3& origin = {0, 0, 0});
```

Comment:
```suggestion
                           const Mn::Vector3& origin = {});
```

is enough, same below